src/modules/tracker.js

```diff
@@ -277,7 +278,7 @@ class Tracker {
    * @param {string} [parameters.tr] - Trigger used to select the item (click, etc.)
    * @param {string} [parameters.itemId] - Item id of the selected item
    * @param {string} [parameters.variationId] - Variation id of the selected item
-   * @param {string} [parameters.groupId] - Group identifier of selected item
+   * @param {string} [parameters.groupId] - Group identifier of the group to search within if a user elected to search in a group, i.e. "Pumpkin in Canned Goods"
```

Comment:
Just a suggestion: Group identifier of the group to search within. Only required if searching within a group, i.e. "Pumpkin in Canned Goods"

Feels like we should emphasize its very optional

Reply:
https://github.com/Constructor-io/constructorio-client-javascript/pull/338/commits/a87406c6ace42cc3a0c12d27b5bfa7197d3be4f9